docs/getting_started/setting_up_an_api.md

```diff
@@ -102,6 +128,12 @@ ENTRY_INFO_SCHEMAS["structures"] = MyStructureResource.schema
 
 Currently, the reference server is not flexible enough to use custom response classes via configuration only (there is an open issue tracking this [#929](https://github.com/Materials-Consortia/optimade-python-tools/issues/929)), so instead the code will need to be forked and modified for your implementation.
 
+!!! note
+    A similar procedure can be followed for the URL query parameter classes [`EntryListingQueryParams`][optimade.server.query_params.EntryListingQueryParams] and [`SingleEntryQueryParams`][optimade.server.query_params.SingleEntryQueryParams] so that custom query parameters can be defined for your API.
+    The individual API routes then need to be adjusted to use the custom query parameter classes.
```

Comment:
I am not entirely sure whether you meant "routes" or "routers" here in line 133.

Reply:
I think routes makes sense, could be either though!